documentation/develop/choosing-a-firefox-version-for-extension-development.md

```diff
@@ -102,19 +102,19 @@ Suggested development use: You may want to install and test a signed copy of you
 
 This version of Firefox is based on Firefox Beta with additional features designed to assist with web and web extension development.
 
-You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/en-US/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/en-US/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties (generally fewer changes than you need to achieve the same tasks in Nightly). This enables you to test features such as restart behavior, permission prompts, and upgrades.
+You load extensions under development into this version of Firefox using [about:debugging](https://developer.mozilla.org/docs/Tools/about:debugging) or [web-ext](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Getting_started_with_web-ext) and have access to the [addon debugging window](https://developer.mozilla.org/docs/Tools/Browser_Toolbox). You can also install unsigned extensions, after adjusting or setting certain about:config properties (generally fewer changes than you need to achieve the same tasks in Nightly). This enables you to test features such as restart behavior, permission prompts, and upgrades.
```

Comment:
Can you change the link for "web-ext" to `/documentation/develop/getting-started-with-web-ext/`?